projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -14,19 +14,38 @@ import './settings.scss';
 
 export const accessOptions = {
 	everybody: {
+		string: 'everybody',
 		label: __( 'Everybody', 'jetpack' ),
 		info: __( 'Visible to everyone.', 'jetpack' ),
 	},
 	subscribers: {
+		string: 'subscribers',
 		label: __( 'All subscribers', 'jetpack' ),
 		info: __( 'Visible to everyone that subscribes to your site.', 'jetpack' ),
 	},
 	paid_subscribers: {
+		string: 'paid_subscribers',
 		label: __( 'Paid subscribers', 'jetpack' ),
 		info: __( 'Visible to everyone that purchases a paid plan on your site.', 'jetpack' ),
 	},
```

Comment:
Instead of adding a `string` key. What do you think of getting the `accessOptions` by doing `Object.keys( accessOptions )`

Reply:
It doesn't work for the purpose I have for them. I know this is not the best way, but TBH I could not figure out a way around it...